client/client.go

```diff
@@ -1157,7 +1158,7 @@ type SdkClientWriteRequestInterface interface {
 
 type ClientWriteRequest struct {
 	Writes  []ClientWriteRequestTupleKey
-	Deletes []ClientWriteRequestTupleKey
+	Deletes []ClientTupleKeyWithoutCondition
```

Comment:
Do we need to update the README examples (i.e., https://github.com/openfga/go-sdk/blob/c3ee17eb6a0558f99c134fa403ad2d6af4e48144/README.md?plain=1#L464)?

Reply:
oh yeah thanks, seems like they were already slightly out of date.